cmd/webhook/main.go

```diff
@@ -46,39 +40,29 @@ var (
 
 type Options struct {
 	Port            int
-	HealthProbePort int
 }
 
 func main() {
 	flag.IntVar(&options.Port, "port", 8443, "The port the webhook endpoint binds to for validation and mutation of resources")
-	flag.IntVar(&options.HealthProbePort, "health-probe-port", 8081, "The port the health probe endpoint binds to for reporting controller health")
 	flag.Parse()
 
 	config := sharedmain.ParseAndGetConfigOrDie()
 
 	// Register the cloud provider to attach vendor specific validation logic.
 	registry.NewCloudProvider(cloudprovider.Options{ClientSet: kubernetes.NewForConfigOrDie(config)})
 
-	// Liveness handler
-	go func() {
-		mux := http.NewServeMux()
-		mux.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) { w.WriteHeader(http.StatusOK) })
-		log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", options.HealthProbePort), mux))
-	}()
-
 	// Controllers and webhook
 	sharedmain.MainWithConfig(
 		webhook.WithOptions(injection.WithNamespaceScope(signals.NewContext(), system.Namespace()), webhook.Options{
 			Port:        options.Port,
 			ServiceName: "karpenter-webhook",
 			SecretName:  "karpenter-webhook-cert",
 		}),
-		"Karpenter Webhooks",
+		"karpenter.webhooks",
```

Comment:
Looks like this component is for logging purposes. Since we name the webhooks `karpenter-webhook`, maybe we could do `Karpenter-Webhooks`? WDYT?

Reply:
It's corresponds to a configmap, which means that the name must be dns compliant. This was causing noisy errors in the log and logging fallback behavior.